client/my-sites/stats/activity-log-banner/upgrade-banner.jsx

```diff
@@ -11,29 +11,54 @@ import { localize } from 'i18n-calypso';
  */
 import { isJetpackSite } from 'state/sites/selectors';
 import Banner from 'components/banner';
-import { PLAN_PERSONAL, FEATURE_JETPACK_ESSENTIAL } from 'lib/plans/constants';
+import {
+	FEATURE_JETPACK_ESSENTIAL,
+	FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+	PLAN_JETPACK_PERSONAL_MONTHLY,
+	PLAN_PERSONAL,
+} from 'lib/plans/constants';
 
 class UpgradeBanner extends Component {
 	render() {
 		const { translate, isJetpack } = this.props;
-		if ( isJetpack ) {
-			return null;
-		}
-		return (
+		return isJetpack ? (
 			<div className="activity-log-banner__upgrade">
 				<Banner
-					callToAction={ translate( 'More Details' ) }
-					dismissPreferenceName="activity-upgrade-banner-simple"
+					callToAction={ translate( 'Learn more' ) }
+					dismissPreferenceName="activity-upgrade-banner-jetpack"
+					event="activity_log_upgrade_click_wpcom"
+					feature={ FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY }
```

Comment:
Should this not also be `FEATURE_JETPACK_ESSENTIAL` ?

Reply:
As confusing as this sound, that feature is only available to highlight for wpcom plans!